Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM gradle:5.3.1-jdk11-slim
+
+USER root
+COPY . hs-test
+RUN chmod 777 /home/gradle/hs-test
```

Comment:
Better would be to copy with chown as dockerfile supports such an option:
```
COPY --chown=gradle . hs-test
```
Instead of these 4 lines:
```
USER root
COPY . hs-test
RUN chmod 777 /home/gradle/hs-test
USER gradle
```